source/developers/minio-drivers.rst

```diff
@@ -18,43 +18,38 @@ MinIO publishes the following Software Development Kits (SDK):
 4. :ref:`Java <java-sdk>`
 5. :ref:`JavaScript <javascript-sdk>`
 6. :ref:`Python <python-sdk>`
+7. :ref:`C++ <cpp-sdk>`
 
+.. _cpp-sdk:
 
-..
-  C++ repo does not have any releases yet. Once released, unblock this section and add to toctree and numbered list.
-  
-  .. _cpp-sdk:
+C++ (``minio-cpp``)
+-------------------
 
-  C++ (``minio-cpp``)
-  -------------------
 
-  Latest Version
-    |cpp-sdk-version|
```

Comment:
Whoops. Deleted my comment.

@balamurugana how are we versioning minio-cpp + there has not been a release in ~1 year.